makefiles/info-global.inc.mk

```
@@ -143,6 +143,9 @@ info-boards-features-blacklisted:
 info-boards-features-conflicting:
 	@for f in $(BOARDS_FEATURES_CONFLICTING); do echo $${f}; done | column -t
 
+Makefile.ci:
```

```suggestion
.PHONY: Makefile.ci
Makefile.ci:
```

This is needed if you want to be able to update an existing `Makefile.ci`. Otherwise it will skip with `make: 'Makefile.ci' is up to date.`

---

I don't think (but do not know and remember the exact rules) that a file that is actually be produced should be marked PHONY; shouldn't this (on an explicit rerun) rather be manually removed and then rebuilt?

Otherwise, make's behavior of rebuilding any stale includes would trigger, and it'd be rebuilt on *ever* use.

---

I did consider that a feature :wink:

---

Interestingly, with `.PHONY` `make` just eats CPU time without ever doing anything...